update_lunch.py

```diff
@@ -0,0 +1,123 @@
+import urllib2
+import urllib
+import string
+import string
+import logging
+import datetime
+import re
+from google.appengine.ext import ndb
+from HTMLParser import HTMLParser
+
+class MLStripper(HTMLParser):
+    def __init__(self):
+        self.reset()
+        self.fed = []
+    def handle_data(self, d):
+        self.fed.append(d)
+    def get_data(self):
+        return ''.join(self.fed)
+
+class Lunch(ndb.Model):
+    summary = ndb.StringProperty(required=True)
+    # description is a list of lines in the description
+    description = ndb.StringProperty(repeated=True)
+    day = ndb.DateProperty(required=True)
+
+def strip_tags(html):
+    s = MLStripper()
+    s.feed(html)
+    return s.get_data()
+
+def getDateObj(text):
+    pass
+
+def getEvents(lines): # lines is a list of all lines of text in the whole file
```

Comment:
suggest calling this parseEvents; parse implies converting text to objects, which is what you are doing here